### DISCUSSION TITLE
Admin permission removed automaticly

### DISCUSSION THREAD

**Discussion by FloHami** [kimai/kimai#5001] [General]
Hello everyone, I have a problem in Kimai. When I grant a user admin permissions, they are gone again after the user logs in. Do you have any tips on what could be causing this? Thank you very much.

**Comment by kevinpapst** [comment]
You are using LDAP or SAML and have the role sync turned on.

**Comment by FloHami** [comment]
How can i check this settings?
Sorry but its my first Kimai
